src/order/order.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Cart, Order, OrderBase, PostOrder } from "../types";
+
+export class OrderService {
+  orderRepository : OrderBase;
+
+  constructor(orderRepository: OrderBase) {
+    this.orderRepository = orderRepository;
+  }
+
+  createOrder(userId: string, cart: Cart) : Order {
+    const newOrder: PostOrder = {
+      userId,
+      cartId: cart.id,
+      items: cart.items,
+      comments: '',
+      status: 'created'
+    }
+    return this.orderRepository.createOrder(newOrder);
```

Comment:
M: I think you should create a new cart each time an order is created with existing cart

Reply:
Got it!
Just for confirming once an order is created we should mark the cart as 'deleted', right?
On task's description it wasn't described, but it makes sense.

Reply:
Yes, exactly!